docs/src/tutorial-real-code.md

```diff
@@ -81,18 +80,5 @@ Running Kani on this simple starting point will help figure out:
 (If the problem is initially intractable, try `--unwind 1` and see if you can follow the techniques in the previous section to put a bound on the problem.)
 
 Once you've got something working, the next step is to prove more interesting properties than what Kani covers by default.
-You accomplish this by adding new assertions.
-These are not necessarily assertions just in your proof harness: consider also adding new assertions to the code being run.
-These count too!
+You accomplish this by adding new assertions (not just in your harness, but also to the code being run).
 Even if a proof harness has no post-conditions being asserted directly, the assertions encountered along the way can be meaningful proof results by themselves.
-
-
-## Summary
-
-In this section:
```

Comment:
Maybe we should keep a summary?

Reply:
Some sections had it while others did not, so during #700 I removed all of them. I don't think they add too much at the section level, at least with the content we currently have.